src/ApiPort/CommandLineOptions.cs

```diff
@@ -70,9 +70,17 @@ private static ICommandLineOptions ShowHelp(string suppliedCommand = null, bool
                 // TODO: Get invalid parameter (Microsoft.Framework.Configuration currently does not surface this)
                 Program.WriteColorLine($"Invalid parameter passed to {suppliedCommand}", ConsoleColor.Red);
             }
-
-            var codebase = new Uri(typeof(CommandLineOptions).GetTypeInfo().Assembly.CodeBase);
-            var path = Path.GetFileName(codebase.AbsolutePath);
+            
+            var location = typeof(CommandLineOptions).GetTypeInfo().Assembly.Location;
+            var path =
+#if NETCORE
```

Comment:
Can you use FEATURE_ASSEMBLY_LOCATION? I think this is already defined (maybe in a different project, though)